scripts/wrappers/reset.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python3
+import click
+import os
+import subprocess
+import sys
+import time
+
+from common.utils import (
+    get_available_addons,
+    get_current_arch,
+    get_status,
+    snap_data,
+    wait_for_ready,
+    snap_common,
+    is_cluster_locked,
+    exit_if_no_permission,
+    ensure_started,
+    kubeconfig,
+)
+
+
+def exit_if_multinode():
+    """
+    Exit if we cannot get the list of nodes or if we are in a multinode cluster
+    """
+    cmd = f"{os.environ['SNAP']}/kubectl {kubeconfig} get no -o name"
```

Comment:
I would much rather get the list of nodes with `-o json` and parse the JSON output instead of resorting to parsing the text output. This happens in a couple of places in this code.

For now, I am OK with the changes to have this move forward.

Reply:
Lets address this in a followup PR.